build.py

```diff
@@ -178,16 +174,13 @@ def build_package(target, build_native_package):
 
     # prepare executables
     os.chdir(os.path.join(script_path))
-    if sys.version_info < (2, 7):
-        python_executable = get_local_path('cpython',['bin','python2.7'])
-    else:
-        python_executable = sys.executable
-        if target == 'cpython':
-            log.debug('skipping cpython ... current python version {0} >= 2.7'.format(platform.python_version()))
-            # touch file to satisfy make
-            if build_native_package:
-                touch(get_package_filename(target))
-            return
+    python_executable = sys.executable
+    if target == 'cpython':
+        log.debug('skipping cpython ... current python version {0} >= 2.7'.format(platform.python_version()))
+        # touch file to satisfy make
+        if build_native_package:
+            touch(get_package_filename(target))
+        return
```

Comment:
is `cpython` still necessary? can we remove it from externals given all supported OS have at least 2.7 or run on 3?

Reply:
yes... also reasonable.

Reply:
i've removed all references to cpython.

see cpython specific commit.

Reply:
got it